utah-geological-survey.csl

```diff
@@ -135,9 +135,9 @@
           <group prefix=", ">
             <text term="in" font-style="italic"/>
             <names variable="editor translator" prefix=" " suffix="," delimiter=", ">
-              <name and="text" sort-separator=", " initialize-with="." name-as-sort-order="all"/>
+              <name and="text" sort-separator=", " initialize-with=".," name-as-sort-order="all"/>
```

Comment:
`initialize-with=".,"` can't be right. I'm pretty sure this would give names like "Doe, J.,B., and T.,S., Smith".

What you want is to use the "delimiter-precedes-last" attribute (and e.g. set it to "always"). See http://docs.citationstyles.org/en/stable/specification.html#name